README.md

```diff
@@ -20,22 +20,15 @@ It is a part of the [OpenMMLab project](https://github.com/open-mmlab).
 The master branch works with **PyTorch 1.3+**.
 
 <div align="center">
-    <img src="demo/resources/demo_coco.gif" width="600px" alt><br>
-    COCO 17-keypoint pose estimation
-</div>
-<div align="center">
-    <img src="https://user-images.githubusercontent.com/9464825/95552839-00a61080-0a40-11eb-818c-b8dad7307217.gif" width="600px" alt><br>
-
-133-keypoint whole-body pose estimation ([full HD version](https://www.youtube.com/watch?v=pIJpQg8mXUU))
+    <img src="https://openmmlab.oss-cn-hangzhou.aliyuncs.com/mmpose/demo/mmpose_full_demo_v2.1s.gif" width="600px" alt><br>
```

Comment:
The gif is 85M, and is too big for most users.
Suppose on is using cellphone, then his/her data will got some loss just viewing this page.

I suggest post this as a video file, as showed ealier

Reply:
Still trying to find a proper way to embed the demo. Video files cannot be embedded into GitHub markdown.

Reply:
https://github.com/open-mmlab/mmpose/issues/429#issuecomment-858308186